tests/functional/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -197,3 +208,210 @@ def test_add_capacity_pre_upgrade(self, reduce_and_resume_cluster_load):
         Test to add variable capacity to the OSD cluster while IOs running
         """
         add_capacity_test()
+
+
+@turquoise_squad
+@stretchcluster_required
```

Comment:
make sure that this test case executes last in the suite. Otherwise, there is a very high chances that the rest of the tests will be skipped due to the unhealthy ceph status caused by pg rebalance